### ISSUE TITLE
First installation does not compile until you create at least one component

### ISSUE THREAD

**Issue by donmbelembe** [archtechx/airwire#16]
Without having at least one airwire component you can't compile your assets, I struggled almost 4 days until I decided to try to create a component first then I was able to compile my assets

**Comment by uwascan** [comment]
This potential game changer appears to have been abandoned. Currently deep diving the source code to see if a fork is doable.

**Comment by stancl** [comment]
Yes, development is paused right now since we're focusing on finishing other projects.

That said, I'm happy to accept PRs. We'll probably be releasing v1 in Q1 2022

**Comment by uwascan** [comment]
@stancl Thank you for that wonderful update.
